.github/CODEOWNERS

```
@@ -0,0 +1,59 @@
+# Note: later rules override earlier rules.
+
+# Default
+#                                  Duncan   Marcin Karl    Nick     Paweł        "Higher ups"
+*                                  @dcoutts @coot  @karknu @nfrisby @EncodePanda @input-output-hk/consensus-superowners
+
+# Network Team
+
+docs                               @dcoutts @coot @karknu @njd42 @peterwthomson
+
+io-sim                             @dcoutts @coot
+io-sim-classes                     @dcoutts @coot
+
+network-mux                        @dcoutts @coot @karknu
+ntp-client                         @coot
+typed-protocols                    @dcoutts @coot
+typed-protocols-examples           @dcoutts @coot
+Win32-network                      @dcoutts @coot
+
+ouroboros-network-framework        @dcoutts @coot @karknu
+ouroboros-network                  @dcoutts @coot @karknu
+ouroboros-network/wireshark-plugin @karknu
+ouroboros-network-testing          @dcoutts @coot @karknu
+
+cardano-client                     @coot
+
+scripts/haddocs.sh                 @coot
+scripts/install.ps1                @coot
+scripts/test.sh                    @coot
+
+# Consensus Team
+
+ouroboros-consensus/               @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-test/          @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-mock/          @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-mock-test/     @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-byron/         @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-byronspec/     @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-byron-test/    @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-shelley/       @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-shelley-test/  @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-cardano/       @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-cardano-test/  @nfrisby @input-output-hk/consensus-superowners
+
+scripts/buildkite/check-stylish.sh @nfrisby @input-output-hk/consensus-superowners
+
+# CICD
+
+bors.toml                          @input-output-hk/devops @input-output-hk/consensus-superowners
+nix/                               @input-output-hk/devops @input-output-hk/consensus-superowners
+*.nix                              @input-output-hk/devops @input-output-hk/consensus-superowners
+cabal.project.local.ci.windows     @input-output-hk/devops @input-output-hk/consensus-superowners
```

```suggestion
cabal.project.local.ci.windows     @coot @input-output-hk/devops @input-output-hk/consensus-superowners
```
I wrote that file :)

---

Fixed, thanks. I left devops in because of the lifting John Ky has been doing.

Feel free to remove devops if you think we should.

Actually, we could create another "secret" team devops-secret if we want them to have access without receiving all the notifications /shrug.